src/content/manifest/hybrid-development.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Hybrid development
+description: Hybrid development allows you to develop and run a service locally while the other components of your app run in the cloud.
+sidebar_label: Hybrid development
+id: hybrid-development
+---
+
+import Image from '@theme/Image';
+
+Okteto offers two different approaches to service development: `sync` or `hybrid` mode. This mode is configured in your Okteto manifest and affects the behavior of [`okteto up`](/reference/cli/#up).
+
+`Sync` is the default mode and changes made to the local service are synchronized with the remote development container deployed in the cluster. Alternatively, `hybrid` mode enables you to run the service in development locally while keeping the rest of the application components in the cloud.
+
+### Deciding which mode to use
+
+There are situations when running a service locally is faster than synchronizing code with a remote development container. For example: when doing frontend-only development or when working in an IDE-heavy context (e.g. Java/Spring) where the IDE may be pre-configured to run tests, start services, and connect debuggers. 
+
+When using Okteto's `hybrid` mode you can maintain your local development experience for the specific service you're working on while still leveraging the benefits of hosting the rest of your application's components in the remote development container.
+
+> Using `hybrid` development mode breaks the replicability between operating systems and increases latency when accessing the rest of the services in the cluster. 
```

Comment:
```suggestion
> Using `hybrid` might increase latency when accessing the rest of the services in the cluster. 
```
I would remove the mention to applicability. This can be solved by running `okteto up` inside a local container